File: Core-Sample/iOS/Siminov/Core/Sample/Model/BookShopMapping.m
//
//  Lession.m
//  core-sample
//
//  Created by user on 13/04/15.
//  Copyright (c) 2015 Siminov. All rights reserved.
//

#import "BookShopMapping.h"

@implementation BookShopMapping

-(Book *)getBook {
    return book;
}

-(void)setBook:(Book *)bk {
    book = bk;
}


-(Shop *)getShop {
    return shop;
}

-(void)setShop:(Shop *)shp {
    shop = shp;
}


@end

File: Core-Sample/Android/CoreSample/src/main/java/siminov/core/sample/model/BookShopMapping.java
/**
 * [SIMINOV FRAMEWORK]
 * Copyright [2014-2016] [Siminov Software Solution LLP|support@siminov.com]
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 **/

package siminov.core.sample.model;

import siminov.core.database.Database;

public class BookShopMapping extends Database {

    //Table Name
    transient public static final String TABLE_NAME = "BOOK_SHOP_MAPPING";

    //Variable Names
    private Book book = null;
    private Shop shop = null;


    //Method Names

    public Book getBook() {
        return this.book;
    }

    public void setBook(Book book) {
        this.book = book;
    }

    public Shop getShop() {
        return this.shop;
    }

    public void setShop(Shop shop) {
        this.shop = shop;
    }
}
